code/modules/client/verbs/autobunker.dm

```diff
@@ -0,0 +1,40 @@
+/client/verb/bunker_auto_authorize()
+	set name = "Auto Authorize Panic Bunker"
+	set desc = "Authorizes your account in the panic bunker of any servers connected to this function."
+	set category = "OOC"
+	
+	var/static/lastuse = 0
```

Comment:
This way, one guy could perpetually run the verb every 5 seconds and deny it for everyone else.

Reply:
@Ghommie it's logged + notify admin'd, i dont' want people able to send more than one topic every once in a while

Reply:
well eh alternative is a client var and i'm not sure if we want that kind of bloat